demo/App/DemoPage.jsx

```diff
@@ -0,0 +1,140 @@
+import React, {Component} from 'react';
+import {safeLoad, dump} from 'js-yaml';
+import {Button, Row, Col, Card} from 'antd';
+
+import 'antd/lib/row/style';
+import 'antd/lib/col/style';
+import 'antd/lib/card/style';
+
+import SchemaEditor from '../components/SchemaEditor';
+import {Form} from '../../src';
+
+export default class DemoPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeTab: 'schema',
+      data: this.props.formData,
+      schema: this.props.schema,
+      uiSchema: this.props.uiSchema,
+    };
+  }
+  handlerSchemaChange = schema => {
+    try {
+      this.setState({schema: safeLoad(schema)});
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  handlerUISchemaChange = uiSchema => {
+    try {
+      this.setState({uiSchema: safeLoad(uiSchema)});
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  handlerDataChange = data => {
+    try {
+      this.setState({data: safeLoad(data)});
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  handleSubmit = () => {
+    this.setState({data: this.form.value});
+  };
+  handleValid = () => {
+    this.form.isValid;
+  };
+
+  handleTabChange = key => {
+    this.setState({activeTab: key});
+  };
+
+  render() {
+    const {schema, uiSchema, data} = this.state;
+
+    const tabs = {
+      schema: (
+        <SchemaEditor
+          key="schema"
+          value={dump(schema)}
+          onChange={this.handlerSchemaChange}
+        />
+      ),
+      uiSchema: (
+        <SchemaEditor
+          key="uiSchema"
+          value={dump(uiSchema)}
+          onChange={this.handlerUISchemaChange}
+        />
+      ),
+    };
+    const tabList = [
+      {
+        key: 'schema',
+        tab: 'Schema definition',
+      },
+      {
+        key: 'uiSchema',
+        tab: 'UI schema definition',
+      },
+    ];
+
+    return (
+      <Row gutter={8} justify="space-around">
+        <Col xl={8} lg={12} md={24}>
+          <Card tabList={tabList} onTabChange={this.handleTabChange}>
```

Comment:
Please add `activeTabKeys` prop (`activeTabKey={this.state.activeTab}`) to highlight active tab after opening new example.